bindings/pyroot/pythonizations/python/ROOT/_numbadeclare.py

```diff
@@ -68,7 +85,8 @@ def get_numba_type(t):
                 'unsigned int': nb.uint32,
                 'long': nb.int64,
                 'unsigned long': nb.uint64,
-                'bool': nb.boolean
+                'bool': nb.boolean,
+                'char': nb.int8
```

Comment:
I'd separate this change in another commit since it is not strictly related to the support of shorthand aliases.

Reply:
I will remove it, since i realize `char` is not a numeric type and hence this change seems unrelated.